<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="33" passed="29" failed="2" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-03-27T09:25:23Z" name="Default suite" finished-at="2021-03-27T09:26:05Z" duration-ms="41110">
    <groups>
    </groups>
    <test started-at="2021-03-27T09:25:23Z" name="Default test" finished-at="2021-03-27T09:26:05Z" duration-ms="41110">
      <class name="com.crm.qa.tests.LoginTests">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:24Z" name="setup" finished-at="2021-03-27T14:55:35Z" duration-ms="11101" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Build1URL_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="Build1URL_Title" finished-at="2021-03-27T14:55:35Z" duration-ms="409" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Build1URL_Title -->
        <test-method signature="Build1_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="Build1_Title" finished-at="2021-03-27T14:55:35Z" duration-ms="41" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Build1_Title -->
        <test-method signature="Build2_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="Build2_Title" finished-at="2021-03-27T14:55:35Z" duration-ms="27" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Build2_Title -->
        <test-method signature="Email_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="Email_Title" finished-at="2021-03-27T14:55:35Z" duration-ms="50" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Email_Title -->
        <test-method signature="Email_URL()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="Email_URL" finished-at="2021-03-27T14:55:35Z" duration-ms="64" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Email_URL -->
        <test-method signature="HotelImage1()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="HotelImage1" finished-at="2021-03-27T14:55:35Z" duration-ms="94" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HotelImage1 -->
        <test-method signature="HotelImage2()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="HotelImage2" finished-at="2021-03-27T14:55:35Z" duration-ms="54" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HotelImage2 -->
        <test-method signature="HotelImage3()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="HotelImage3" finished-at="2021-03-27T14:55:35Z" duration-ms="102" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HotelImage3 -->
        <test-method signature="HotelImage4()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:35Z" name="HotelImage4" finished-at="2021-03-27T14:55:36Z" duration-ms="59" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HotelImage4 -->
        <test-method signature="InvalideloginTest1()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:36Z" name="InvalideloginTest1" finished-at="2021-03-27T14:55:37Z" duration-ms="1131" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InvalideloginTest1 -->
        <test-method signature="Password_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:37Z" name="Password_Title" finished-at="2021-03-27T14:55:37Z" duration-ms="37" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Password_Title -->
        <test-method signature="Signup()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:37Z" name="Signup" finished-at="2021-03-27T14:55:40Z" duration-ms="3621" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Signup -->
        <test-method signature="Signup_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:40Z" name="Signup_Title" finished-at="2021-03-27T14:55:40Z" duration-ms="46" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Signup_Title -->
        <test-method signature="Username_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:40Z" name="Username_Title" finished-at="2021-03-27T14:55:40Z" duration-ms="36" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Username_Title -->
        <test-method signature="ValideloginTest()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:40Z" name="ValideloginTest" finished-at="2021-03-27T14:55:42Z" duration-ms="1630" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValideloginTest -->
        <test-method signature="build_title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:42Z" name="build_title" finished-at="2021-03-27T14:55:42Z" duration-ms="45" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- build_title -->
        <test-method signature="footer_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:42Z" name="footer_Title" finished-at="2021-03-27T14:55:42Z" duration-ms="40" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- footer_Title -->
        <test-method signature="forgotPass()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:42Z" name="forgotPass" finished-at="2021-03-27T14:55:43Z" duration-ms="989" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- forgotPass -->
        <test-method signature="forgotten_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:43Z" name="forgotten_Title" finished-at="2021-03-27T14:55:43Z" duration-ms="45" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- forgotten_Title -->
        <test-method signature="hotel_URL()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:43Z" name="hotel_URL" finished-at="2021-03-27T14:55:43Z" duration-ms="36" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hotel_URL -->
        <test-method signature="loginPageTitleTest()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:43Z" name="loginPageTitleTest" finished-at="2021-03-27T14:55:43Z" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTitleTest -->
        <test-method signature="loginTitle()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:43Z" name="loginTitle" finished-at="2021-03-27T14:55:43Z" duration-ms="34" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTitle -->
        <test-method signature="resetPassword()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:43Z" name="resetPassword" finished-at="2021-03-27T14:55:45Z" duration-ms="1575" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- resetPassword -->
        <test-method signature="validateHeaderImg()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:45Z" name="validateHeaderImg" finished-at="2021-03-27T14:55:45Z" duration-ms="55" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateHeaderImg -->
        <test-method signature="validateImage()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:45Z" name="validateImage" finished-at="2021-03-27T14:55:45Z" duration-ms="60" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateImage -->
        <test-method signature="validateLOgo1()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:45Z" name="validateLOgo1" finished-at="2021-03-27T14:55:45Z" duration-ms="45" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLOgo1 -->
        <test-method signature="welcome_Title()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:45Z" name="welcome_Title" finished-at="2021-03-27T14:55:45Z" duration-ms="42" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- welcome_Title -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.crm.qa.tests.LoginTests@5fd4f8f5]" started-at="2021-03-27T14:55:45Z" name="teardown" finished-at="2021-03-27T14:55:46Z" duration-ms="796" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.crm.qa.tests.LoginTests -->
      <class name="com.crm.qa.tests.ForgotTest">
        <test-method is-config="true" signature="set()[pri:0, instance:com.crm.qa.tests.ForgotTest@696da30b]" started-at="2021-03-27T14:55:46Z" name="set" finished-at="2021-03-27T14:55:52Z" duration-ms="6144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- set -->
        <test-method signature="GoBack()[pri:0, instance:com.crm.qa.tests.ForgotTest@696da30b]" started-at="2021-03-27T14:55:52Z" name="GoBack" finished-at="2021-03-27T14:55:52Z" duration-ms="2" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.crm.qa.pages.ForgotPage.GoBackLoginPage(ForgotPage.java:138)
	at com.crm.qa.tests.ForgotTest.GoBack(ForgotTest.java:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GoBack -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.crm.qa.tests.ForgotTest@696da30b]" started-at="2021-03-27T14:55:52Z" name="teardown" finished-at="2021-03-27T14:55:53Z" duration-ms="597" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.crm.qa.tests.ForgotTest -->
      <class name="com.crm.qa.tests.SignUpTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.crm.qa.tests.SignUpTest@4e7912d8]" started-at="2021-03-27T14:55:53Z" name="setup" finished-at="2021-03-27T14:55:59Z" duration-ms="5973" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="m()[pri:0, instance:com.crm.qa.tests.SignUpTest@4e7912d8]" started-at="2021-03-27T14:55:59Z" name="m" finished-at="2021-03-27T14:55:59Z" duration-ms="7" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ytdruyhgrf] but found [Adactin.com - New User Registration]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ytdruyhgrf] but found [Adactin.com - New User Registration]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.crm.qa.tests.SignUpTest.m(SignUpTest.java:46)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m -->
        <test-method signature="register(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.crm.qa.tests.SignUpTest@4e7912d8]" started-at="2021-03-27T14:55:59Z" name="register" data-provider="getData1" finished-at="2021-03-27T14:55:59Z" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.io.FileNotFoundException: C:\Users\Hp\eclipse-workspace\FreeCRMTest\src\main\java\com\crm\qa\testdata\testdata.xlsx (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: C:\Users\Hp\eclipse-workspace\FreeCRMTest\src\main\java\com\crm\qa\testdata\testdata.xlsx (The system cannot find the path specified)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:162)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.FileNotFoundException: C:\Users\Hp\eclipse-workspace\FreeCRMTest\src\main\java\com\crm\qa\testdata\testdata.xlsx (The system cannot find the path specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:220)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:158)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:113)
	at com.crm.qa.util.TestUtil.fileRead(TestUtil.java:31)
	at com.crm.qa.tests.SignUpTest.getData(SignUpTest.java:64)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- register -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.crm.qa.tests.SignUpTest@4e7912d8]" started-at="2021-03-27T14:55:59Z" name="teardown" finished-at="2021-03-27T14:55:59Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.crm.qa.tests.SignUpTest -->
      <class name="com.crm.qa.tests.HomeTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.crm.qa.tests.HomeTest@53976f5c]" started-at="2021-03-27T14:55:59Z" name="setup" finished-at="2021-03-27T14:56:04Z" duration-ms="5691" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="locationDropDown1(java.lang.String)[pri:0, instance:com.crm.qa.tests.HomeTest@53976f5c]" started-at="2021-03-27T14:56:04Z" name="locationDropDown1" data-provider="getData1" finished-at="2021-03-27T14:56:04Z" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.io.FileNotFoundException: C:\Users\Hp\eclipse-workspace\FreeCRMTest\src\main\java\com\crm\qa\testdata\testdata.xlsx (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: C:\Users\Hp\eclipse-workspace\FreeCRMTest\src\main\java\com\crm\qa\testdata\testdata.xlsx (The system cannot find the path specified)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:162)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.FileNotFoundException: C:\Users\Hp\eclipse-workspace\FreeCRMTest\src\main\java\com\crm\qa\testdata\testdata.xlsx (The system cannot find the path specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:220)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:158)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:113)
	at com.crm.qa.util.TestUtil.fileRead(TestUtil.java:31)
	at com.crm.qa.tests.HomeTest.getData(HomeTest.java:47)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- locationDropDown1 -->
        <test-method signature="validateCheckInDate()[pri:0, instance:com.crm.qa.tests.HomeTest@53976f5c]" started-at="2021-03-27T14:56:04Z" name="validateCheckInDate" finished-at="2021-03-27T14:56:04Z" duration-ms="179" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateCheckInDate -->
        <test-method signature="validateCheckOutDate()[pri:0, instance:com.crm.qa.tests.HomeTest@53976f5c]" started-at="2021-03-27T14:56:04Z" name="validateCheckOutDate" finished-at="2021-03-27T14:56:05Z" duration-ms="165" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateCheckOutDate -->
      </class> <!-- com.crm.qa.tests.HomeTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
